.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+<!--
+
+Thanks for filing an issue! If this is a question or feature request, just delete
+everthing here and write out the request, providing as much context as you can.
+
+-->
+
+### What version of Go (`go version`) and `dep` are you using?
```

Comment:
Since we don't have `dep version`, let's ask people to run `git describe --tags`.